docs/documentation.toml

```diff
@@ -287,6 +287,35 @@ description = "Throws an error if the prompt is canceled, instead of returning a
 name = "multiline"
 description = "If set to true, the input field will be a multiline textarea"
 
+[tp.system.functions.multiselect]
```

Comment:
This is a tad bit nitpicky, but it may make sense to have this function be called `multi_suggester` to match the naming of the `suggester` function. Which is unfortunate, as I think I'd like `select` and `multiselect` better, but I don't think I want to backtrack on the `suggester`.

Reply:
Sure, it's no problem I will rename it to `multi_suggester` to make more consistent.